pkg/config/fulcio_config_test.go

```diff
@@ -0,0 +1,76 @@
+// Copyright 2024 The Sigstore Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+//go:build !hermetic
+
+package config
+
+import (
+	"os"
+	"path/filepath"
+	"runtime"
+	"testing"
+
+	"gopkg.in/yaml.v3"
+)
+
+type FulcioConfigMap struct {
+	Data map[string]string `yaml:"data,omitempty"`
+}
+
+// It tests that the config/fulcio-config.yaml is properly parsable
+func TestLoadFulcioConfig(t *testing.T) {
+	_, path, _, _ := runtime.Caller(0)
+	basepath := filepath.Dir(path)
+	b, err := os.ReadFile(basepath + "/../../config/identity/config.yaml")
+	if err != nil {
+		t.Errorf("read file: %v", err)
+	}
+
+	cfg := FulcioConfigMap{}
```

Comment:
Do we need this config map, or can we just call `Read(b)`?

Reply:
we didn't! removing